package.json

```diff
@@ -20,7 +20,8 @@
   },
   "dependencies": {
     "ember-cli-babel": "^6.6.0",
-    "ember-config-service": "^0.2.0"
+    "ember-config-service": "^0.2.0",
```

Comment:
I think `ember-config-service` now shouldn't be needed anymore, right? :)

Reply:
not unless you want to keep compatibility going back to older versions of ember which didn't have config lookup the way we do now